examples/ssbvm_mixture.py

```diff
@@ -0,0 +1,297 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+r"""
+Example: Sine-skewed sine (bivariate von Mises) mixture
+=======================================================
+
+This example models the dihedral angles that occur in the backbone of a protein as a mixture of skewed
+directional distributions. The backbone angle pairs, called :math:`\phi` and :math:`\psi`, are a canonical
+representation for the fold of a protein. In this model, we fix the third dihedral angle (omega) as it usually only
+takes angles 0 and pi radian, with the latter being the most common. We model the angle pairs as a distribution on
+the torus using the sine distribution [1] and break point-wise (toroidal) symmetry using sine-skewing [2].
+
+.. image:: ../_static/img/examples/torus_top.png
```

Comment:
Maybe `ssbvm_mixture_torus_top.png` is a better name.

Reply:
Make sense; I'll update it.